tests/ballerina-compiler-api-test/src/test/resources/test-src/symbol_at_cursor_basic_test.bal

```diff
@@ -164,3 +164,13 @@ function test(string name, int age) {
         [foo()]: "FOO"
     };
 }
+
+xmlns "foo" as ns;
+
+int indx = 0;
+
+function testXMLStepExpr() {
+    xml x1 = xml `<ns:root><ns:child></ns:child></ns:root>`;
+    xml x2 = x1/<ns:child>[indx];
+    string s = x1/**/<ns:child>.toString();
```

Comment:
These tests aren't needed. Only have to enable the tests in `SymbolsInXMLNavigationExprsTest`

Reply:
Updated.